packages/landing/static/examples/index.openlayers.wmts.3857.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>OpenLayers WMTS Basemaps Demo</title>
+    <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/css/ol.css"
+        type="text/css"
+    />
+    <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/build/ol.js"></script>
+  </head>
+
+  <body>
+    <div id="map" style="width: 800px; height: 600px;"></div>
+    <script>
+     const apiKey = localStorage.getItem('api-key');
+     var attributions =
+         '<a href="https://www.linz.govt.nz/data/linz-data/linz-basemaps/data-attribution">LINZ CC BY 4.0 © Imagery Basemap contributors</a>';
+     const projstr = "EPSG:3857";
+     const startPos = [19467552, -5074414];
```

Comment:
could make these short hands `const center = [..]` or just add them to the view itself?